packages/preview/splendid-mdpi/0.1.0/typst.toml

```diff
@@ -0,0 +1,28 @@
+# for a description of available keys, see https://github.com/typst/packages/?tab=readme-ov-file#package-format
+
+[package]
+name = "splendid-mdpi"
+version = "0.1.0"
+entrypoint = "src/lib.typ"
+authors = ["James R. Swift"]
+license = "Unlicense"
+description = "An MDPI-style paper template to publish at conferences and journals"
+repository = "https://github.com/JamesxX/splendid-mdpi"
+keywords = ["paper", "article", "journal", "conference"]
+categories = ["paper"]
+exclude = [
+	".github",
+	"docs",
+	"scripts",
+	"tests",
+	".typstignore",
+	"Justfile",
```

Comment:
I didn't notice this before, but if you have two thumbnails, the `thumbnails` directory should be excluded. The real thumbnail is excluded by default, but the second one wouldn't be.

Reply:
Note: In https://github.com/typst/packages/pull/809, it's correct, but in https://github.com/typst/packages/pull/807 it is missing as well.